### ISSUE TITLE
Motion and HSV recording

### ISSUE THREAD

**Issue by mihailescu2m** [seydx/homebridge-camera-ui#568]
I have camera with hardware motion sensor, exposed through MQTT.
I enabled HSV, and want to use recording on person detection.
Currently, the only way to get that working is to set videoanalysis.active = true.
Is there a way to have HSV detecting motion using MQTT camera sensor instead of videoanalysis? My understanding is that object detection is done by Apple in the cloud, so I think videoanalysis only does motion detection and not object recognition?

**Comment by utezduyar** [comment]
I think object detection is done on your network and object classification is done in the cloud.

Ref: https://support.apple.com/guide/security/camera-security-sec525461d19/web
`HomeKit decrypts the audio and video streams on the home hub and analyzes the video frames locally for any significant event.`

**Comment by mihailescu2m** [comment]
Yes by object detection and object recognition I meant classification, vs motion detection.
So the question is the same: Is there a way to have HSV detecting motion using MQTT camera sensor instead of videoanalysis?

**Comment by utezduyar** [comment]
@mihailescu2m Does the mqtt client functionality work and trigger HSV classification even when the video analysis is off?

**Comment by mihailescu2m** [comment]
@utezduyar yes, and my cpu usage now is low

**Comment by utezduyar** [comment]
There is no MQTT broker support in camera.ui right, only client? You must be using an external broker.

**Comment by Rahulsharma0810** [comment]
@mihailescu2m can you share the conf for onvif2mqtt, I am trying to implement this for my camera.

**Comment by mihailescu2m** [comment]
@Rahulsharma0810 

```
$ cat config.yml
mqtt:
  host: <mqtt_broker_ip>
  port: 1883
onvif:
  - name: <camera1_name>
    hostname: <camera1_ip>
    port: <camera1_onvif_port>
    username: <camera1_username>
    password: <camera1_password>
  - name: camera2
    hostname: 192.168.0.22
    port: 2020
    username: camera2user
    password: camera2pass
```

**Comment by Rahulsharma0810** [comment]
@mihailescu2m thanks for it can you share your whole docker-compose setup, in my case even though both services are up, it is not detecting any event.

**Comment by mihailescu2m** [comment]
@Rahulsharma0810 I don't use docker-compose for this, just started a container with onvif2mqtt ...

**Comment by Rahulsharma0810** [comment]
Thanks for the quick reply, I made it work, can you share your cameraui mqtt conf.

Thanks @mihailescu2m

**Comment by mihailescu2m** [comment]
```
      "mqtt": {
          "motionTopic": "onvif2mqtt/<camera1_name>/motion",
          "motionMessage": "ON",
          "motionResetTopic": "onvif2mqtt/<camera1_name>/motion",
          "motionResetMessage": "OFF"
      },
```

**Comment by mkz212** [comment]
Probable solution: #855
